Main.java

```diff
@@ -86,120 +128,86 @@ static public void putSignQuestions(Player player) { // bo Main jest metodą sta
         printTable();
     }
 
-    static public boolean checkFreeField(){
-        if (gameTable[columnNumber][rawNumber] == 'O' || gameTable[columnNumber][rawNumber] == 'X' ) {
+    static public boolean checkFreeField() {
+        if (gameTable[rawNumber][columnNumber] != '_') {
             System.out.println("This field is taken! Please fill an empty field.");
             freeFieldVar = false;
-        }
-        else
+        } else
             freeFieldVar = true;
         return freeFieldVar;
     }
 
-    static public boolean checkWin(Player player) {  // jak to sprawdzić inaczej? pętla w pętli?
-        // ponieważ teraz kod nie jest rozszerzalny, co w przypadku tablicy 100x100 itp. ?
-            char s = player.getSymbol();
-            boolean playerWinner = false;
-            if (
-
-                gameTable[0][0] == s && gameTable[1][0] == s && gameTable[2][0] == s ||
-                gameTable[0][1] == s && gameTable[1][1] == s && gameTable[2][1] == s ||
-                gameTable[0][2] == s && gameTable[1][2] == s && gameTable[2][2] == s ||
-                gameTable[0][0] == s && gameTable[0][1] == s && gameTable[0][2] == s ||
-                gameTable[1][0] == s && gameTable[1][1] == s && gameTable[1][2] == s ||
-                gameTable[2][0] == s && gameTable[2][1] == s && gameTable[2][2] == s ||
-                gameTable[0][0] == s && gameTable[1][1] == s && gameTable[2][2] == s ||
-                gameTable[0][2] == s && gameTable[1][1] == s && gameTable[2][0] == s
-
-
-                    ) {
-                System.out.println("Player 1 " + player.getName() + "is a WINNER");
+    static public boolean checkWin(Player player) {
+        char s = player.getSymbol();
```

Comment:
s is not a good name for the variable - single letter variables are allowed only as counters in loops